redash/settings.py

```diff
@@ -73,6 +73,8 @@ def all_settings():
 CELERY_BROKER = os.environ.get("REDASH_CELERY_BROKER", REDIS_URL)
 CELERY_BACKEND = os.environ.get("REDASH_CELERY_BACKEND", CELERY_BROKER)
 CELERY_TASK_RESULT_EXPIRES = int(os.environ.get('REDASH_CELERY_TASK_RESULT_EXPIRES', 3600))
+CELERYD_PREFETCH_MULTIPLIER = int(os.environ.get('REDASH_CELERYD_PREFETCH_MULTIPLIER', 4))
+CELERY_ACKS_LATE = parse_boolean(os.environ.get('REDASH_CELERY_ACKS_LATE', "false"))
```

Comment:
I've looked at `rq` for personal projects and liked what I saw, but never actually tried it. Also, I'm worried I'd miss just enough of celery's extra functionality that I'd regret it. 

At work, it really depends on the need... rq doesn't have enough tuning knobs for many of our more advanced use cases, celery hits a nice sweet spot when you need more than a worker queue but less than migrating the service into a full-blown kafka pipeline. The other big problem though we run into with celery is that we have some tasks where we can't lose the data for, but celery's best backend is redis, where getting enough durability guarantees require a fair bit of operational overhead. And a couple of those services also are often at risk of having redis run out of memory (even on very large boxes) because celery doesn't support redis-cluster... which forces us to load balance at a different layer of the stack where it's often more of a pain operationally... so I'd say we have a like/dislike relationship with celery that isn't strong enough either direction to be a love/hate relationship.